src/config.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+import path from 'path';
+
+import requireUncached from 'require-uncached';
+
+import {createLogger} from './util/logger';
+import {UsageError} from './errors';
+
+const log = createLogger(__filename);
+
+type ApplyConfigToArgvParams = {|
+  argv: Object,
+  configObject: Object,
+  defaultValues: Object,
+|};
+
+type ParseConfigParams = {|
+  sourceDir: string,
+  configFileName: string,
+|};
+
+export function applyConfigToArgv({
+  argv,
+  configObject,
+  defaultValues = {},
+}: ApplyConfigToArgvParams): Object {
+  for (const option in configObject) {
+    if (argv[option] && defaultValues[option]) {
+      const wasValueSetOnCLI = argv[option] !== defaultValues[option];
```

Comment:
If you change this to the following then you won't need to wrap it in the if statement up above:
````js
const wasValueSetOnCLI = argv[option] && argv[option] !== defaultValues[option];
````